CollAction/Controllers/ProjectsController.cs

```diff
@@ -364,12 +365,35 @@ public ViewResult StartInfo()
         }
 
         [HttpGet]
-        public async Task<JsonResult> GetTileProjects(int? categoryId, int? locationId)
+        public async Task<JsonResult> GetTileProjects(int? categoryId, int? locationId, string statusId)
```

Comment:
locationId is still not needed as a parameter, location-wise search will probably take the form of, longitude/latitude/radius.